pkg/proxy/router/router.go

```diff
@@ -663,6 +661,11 @@ func (s *Server) RegisterAndWait(wait bool) {
 
 func NewServer(conf *Conf) *Server {
 	log.Infof("start with configuration: %+v", conf)
+
+	f := func(addr string) (*redisconn.Conn, error) {
+		return redisconn.NewConnectionWithSize(addr, conf.NetTimeout, 16*1024, 16*1024)
```

Comment:
can you set 16*1024 as a const and explain why you choose this value? 

This would be easier for other people to look through the code and help you profiling when necessary.